main.go

```diff
@@ -86,18 +87,20 @@ func main() {
 		os.Exit(1)
 	}
 	options := ctrl.Options{
-		Scheme:             scheme,
-		MetricsBindAddress: metricsAddr,
-		Port:               9443,
```

Comment:
https://github.com/openshift/cincinnati-operator/blob/8d09abc9fa733dea12bb56a12b96d28b36685257/vendor/sigs.k8s.io/controller-runtime/pkg/manager/manager.go#L230-L232

It seems we do not use the webhook at all.